modules/promotion/src/PromotionStorage.php

```diff
@@ -24,8 +24,27 @@ public function loadValid(OrderTypeInterface $order_type, StoreInterface $store)
     if (empty($result)) {
       return [];
     }
+
+    $usages_query = $this->database->select('commerce_promotion_usage', 'cpu');
+    $usages_query->condition('promotion_id', array_keys($result), 'IN');
+    $usages_query->addField('cpu', 'promotion_id');
+    $usages_query->addExpression("COUNT(promotion_id)", 'count');
```

Comment:
Let's call the PromotionUsage method: getUsageMultiple(array $promotions, array $coupons)